qiskit/providers/ibmq/credentials/configrc.py

```diff
@@ -33,17 +34,18 @@
 
 def read_credentials_from_qiskitrc(
         filename: Optional[str] = None
-) -> Dict[HubGroupProject, Credentials]:
+) -> Tuple[Dict[HubGroupProject, Credentials], str]:
     """Read a configuration file and return a dictionary with its contents.
 
     Args:
         filename: Full path to the configuration file. If ``None``, the default
             location is used (``$HOME/.qiskit/qiskitrc``).
 
     Returns:
-        A dictionary with the contents of the configuration file, in the
-        ``{credential_unique_id: Credentials}`` format. The dictionary is
-        empty if the input file does not exist.
+        A tuple containing the found credentials, if any, and the default
+        provider stored, if specified in the configuration file. The format
+        for the found credentials is ``{credentials_unique_id: Credentials}``,
+        whereas the format for the default provider is ``<hub_name>/<group_name>/<project_name>``.
```

Comment:
One reason was to be able to use the string if an exception occurs, to show the user the string they saved in the configuration file. But, I could refactor the exceptions to take this into account, it might make things cleaner in the end as well. Thanks!